cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -102,6 +102,9 @@ share
     fee                 DbLovelace          sqltype=lovelace
     deposit             Int64                                   -- Needs to allow negaitve values.
     size                Word64              sqltype=uinteger
+
+    invalidBefore       Word64 Maybe        sqltype=uinteger
+    invalidAfter        Word64 Maybe        sqltype=uinteger
```

Comment:
Great. We just need to confirm with the ledger folks if they're inclusive or exclusive bounds.

If they're exclusive (as a first reading of the code suggests), then we should swap the names:
* "invalidBefore" to be "validAfter"
* "invalidAfter" to be "validBefore"